OpenSim/Common/C3DFileAdapter.h

```diff
@@ -44,10 +42,13 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
     C3DFileAdapter& operator=(C3DFileAdapter&&)      = default;
     ~C3DFileAdapter()                                = default;
 
+    explicit C3DFileAdapter(const bool& useSIMMColumnOrdering);
```

Comment:
It may make more sense and require fewer assumptions and checks to have a separate export to SIMMMotionFile and then check if the Table has forces, if so, then reorder for SIMM compatibility. In any case the output *.mot* file should have the appropriate SIMM header.